src/app/static/src/app/components/ErrorAlert.vue

```diff
@@ -1,11 +1,63 @@
 <template>
     <div class="pt-1 text-danger">
-        {{message}}
+        <div class="error-message">{{message}}</div>
+        <div v-if="hasTrace" >
+            <a href="#" @click="toggleTrace"><strong>{{traceLinkText}}</strong></a>
```

Comment:
Could style this with up/down carets by conditionally adding "up"/"down" classes to this link and the style rules:
```

a.up::after {
  content: '\25b2';
}

a.down::after {
  content: '\25bc';
}
```